src/resources/views/base/inc/sidebar.blade.php

```diff
@@ -25,10 +25,18 @@
   <script type="text/javascript">
     // Save default sidebar class
     let sidebarClass = (document.body.className.match(/sidebar-(sm|md|lg|xl)-show/) || ['sidebar-lg-show'])[0];
+    let sidebarTransition = value => document.querySelector('.sidebar').style.transition = value || '';
 
     // Recover sidebar state
     let sessionState = sessionStorage.getItem('sidebar-collapsed');
-    if(sessionState) document.body.classList.toggle(sidebarClass, sessionState === '1');
+    if(sessionState) {
+      // disable the transition animation temporarly
```

Comment:
```suggestion
      // disable the transition animation temporarily, so that if you're browsing across
      // pages with the sidebar closed, the sidebar does not flicker into the view
```